streetcrm/admin.py

```diff
@@ -552,8 +551,24 @@ def advanced_search_do(self, request, form, export):
                                "result_count": count_set['result_count']
             }
         else:
+            search_params = []
+            for key, value in data.items():
+                if isinstance(value, datetime):
+                    search_params.append('{} = {}'.format(key, value.strftime('%Y-%m-%d')))
+                elif isinstance(value, str) and len(value):
+                    search_params.append('{} = {}'.format(key, value))
+                elif isinstance(value, bool) and value:
+                    search_params.append(key)
+                elif hasattr(value, '__iter__') and len(value):
+                    search_params.append('{} = {}'.format(key, ','.join([str(v) for v in value])))
+                elif hasattr(value, 'pk'):
+                    search_params.append('{} = {}'.format(key, str(value)))
+
+            search_header = ', '.join(search_params)
+
             results_package = {"form": form,
                                "search_results": results,
+                               "search_header": search_header,
```

Comment:
A better solution might be to keep the correspondence between the form keys and parameters listed in the CSV header row (because that's easy to maintain & is intuitive for other developers encountering the code), but just give the form keys better names, where "better" means "more meaningful to an end user".